src/app/_services/error.service.ts

```diff
@@ -60,7 +60,7 @@ export class ErrorService {
         throwError({
           status: 0,
           error: { errorMessage: this.translate.instant('errorRetryFailed') }
-        })
+        } as HttpErrorResponse)
```

Comment:
The point is that you can now remove the disable lint rule statement.
Also: is this not a solution for the other places where you disabled this lint rule?

Reply:
Of course!  Done.